src/iadmin.cpp

```diff
@@ -1437,24 +1364,6 @@ doCommand( char *cmdToken[], rodsArguments_t* _rodsArgs = 0 ) {
         show_global_quotas(cmdToken[1]);
         return 0;
     }
-    if ( strcmp( cmdToken[0], "mkdir" ) == 0 ) {
-        if ( _rodsArgs->force == True ) {
-            int path_index = 1;
-#ifdef osx_platform
-            path_index = 2;
-#endif
-            generalAdmin( 0, "add", "dir",
-                          cmdToken[path_index], "",
-                          "", "",
-                          "", "",
-                          "", "" );
```

Comment:
Are we mainly interested in just removing `iadmin`'s exposure of this functionality, or the whole feature? The "add" of a "dir" is apparently part of the GeneralAdmin API.

Reply:
Okay. Consider making an issue for this and the "rm" of "dir" from GeneralAdmin API.

Reply:
created as https://github.com/irods/irods/issues/8499